administrator/language/de-DE/plg_fields_sql.ini

```
@@ -8,6 +8,8 @@ PLG_FIELDS_SQL="Felder – SQL"
 PLG_FIELDS_SQL_CREATE_NOT_POSSIBLE="Nur ein Super Benutzer kann ein SQL-Feld erstellen."
 PLG_FIELDS_SQL_LABEL="SQL (%s)"
 PLG_FIELDS_SQL_PARAMS_FORM_LAYOUT_FANCY_SELECT="Erweitertes Auswählen"
+PLG_FIELDS_SQL_PARAMS_HEADER_DESC="Fügt eine Auswahlmöglichkeit ohne Wert an den Anfang der Dropdown-Liste, z.&#8239;B. ' - Artikel wählen - '."
```

```suggestion
PLG_FIELDS_SQL_PARAMS_HEADER_DESC="Fügt eine Auswahlmöglichkeit ohne Wert an den Anfang der Dropdown-Liste hinzu, z.&#8239;B. ' - Beitrag wählen - '."
```

Passt mehr zu unserem Sprachgebrauch

---

Ja, ist aktualisiert